client/templates/selectedProduct/selectedProduct.html

```diff
@@ -0,0 +1,30 @@
+<template name="selectedProduct">
+  <h1>Selected Product:</h1>
+    {{#if Template.subscriptionsReady}}
+      {{#if selectedProduct}}
+        {{#with selectedProduct}}
+          {{>product}}
+        {{/with}}
+      {{else}}
+        Sorry, no product with that code...
+      {{/if}}
+    {{else}}
+      Loading...
+    {{/if}}
+</template>
+
+<template name="product">
+  <h2>{{description}}</h2>
+  <p>Price: ${{price}}</p>
+  <p>Cost: ${{cost}}</p>
+  <p>Look Up Code: {{itemLookUpCode}}</p>
+  <p>Type: {{itemType}}</p>
+  <p>Quantity: {{quantity}}</p>
+  <p>Reorder Point: {{reorderPoint}}</p>
+  <p>Restock Level:{{restockLevel}}</p>
+  <p>Parent Item: {{parentItem}}</p>
+  <p>Extended Description: {{extendedDescription}}</p>
+  <p>Active: {{isActive}}</p>
+  <p>MSRP: {{msrp}}</p>
+  <p>Created at: {{createdAt}}</p>
```

Comment:
Hmm, I dunno.

I mean, thinking about it from the POV of the Sales Clerk who may be using it:
Say a customer calls you up and asks something like "Hey when did you start stocking this item?" (Can't quite think of a good reason why they might do this, but seems plausible) Then you can look it up and see when it was added to the database (which is when, I assume, they would have started stocking it.